packages/shorebird_cli/lib/src/commands/preview_command.dart

```diff
@@ -143,12 +147,51 @@ class PreviewCommand extends ShorebirdCommand {
       return ExitCode.software.code;
     }
 
+    final completeRelease = allReleases.firstWhereOrNull(
```

Comment:
The naming here is a bit confusing. What makes this release "complete"?

Reply:
This release will be the same as `release` above, but will potentially contain more information if `release` contains non-previewable platforms (eg, an unsigned iOS release).